### PR TITLE
Update to sbt-lucuma 0.6

### PR DESCRIPTION
**Pull request by armanbilge**

### CODE REVIEW

File: src/main/scala/lucuma/broadcastchannel/package.scala [hunk 1]
```diff
@@ -13,7 +16,7 @@ package object broadcastchannel {
 
   implicit def ioToOnMessageHandler[T](f: T => IO[Unit]): OnMessageHandler[T] =
     js.ThisFunction.fromFunction2(
-      (_: lucuma.bc.broadcastChannel.broadcastChannelMod.BroadcastChannel[js.Any], x: T) =>
```
Comment (cquiroz) [inline src/main/scala/lucuma/broadcastchannel/package.scala:16]: is `js.Any` replaceable by `Any`?

Comment (armanbilge) [reply src/main/scala/lucuma/broadcastchannel/package.scala:16]: See https://github.com/ScalablyTyped/Converter/issues/351.

Comment (cquiroz) [reply src/main/scala/lucuma/broadcastchannel/package.scala:16]: Cool, thanks

**Review by cquiroz — APPROVED** [review]
LGTM

**Pull request merged**